src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentServer.kt

```diff
@@ -74,8 +91,10 @@ interface CodyAgentServer {
   @JsonRequest("chat/submitMessage")
   fun chatSubmitMessage(params: ChatSubmitMessageParams): CompletableFuture<ExtensionMessage>
 
-  @JsonRequest("chat/models")
-  fun chatModels(params: ChatModelsParams): CompletableFuture<ChatModelsResponse>
+  @JsonRequest("chat/models") fun chatModels(params: IdParam): CompletableFuture<ChatModelsResponse>
+
+  @JsonRequest("chat/export")
+  fun chatExport(params: IdParam): CompletableFuture<ChatHistoryResponse?>
```

Comment:
This does not take an ID any more on the extension side.

Reply:
thanks